common/config/azure-pipelines/npm-publish.yaml

```diff
@@ -10,6 +10,10 @@ steps:
   - template: templates/bump-versions.yaml

```

Comment:
This strategy is a no-go. Rush currently only supports publishing under the original name.